dotnet-client-libraries/YamlDotNet/App/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿using App.Models;
+using App.UseCases;
+using YamlDotNet.Core;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NodeDeserializers;
+using static System.Console;
+
+namespace App;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        // Serialize and Deserialize Basic Object
+
+        var yamlProduct = BasicSerializeAndDeserialize.SerializeProduct(new Product
+        {
+            Id = 1,
+            Name = "Apple",
+            Price = 1.99m
+        });
+        WriteLine(yamlProduct);
```

Comment:
Please change these `WriteLine()` statements to `Console.WriteLine()`.

I know this one's subjective too, and the static `using` directives can really DRY up code, but in this instance having `Console.` on the beginning doesn't make it that much longer, and including it avoids any confusion. Someone may wonder for a minute where the `WriteLine()` method is defined in your code, before realizing that it's actually the .NET one.

```suggestion
        Console.WriteLine(yamlProduct);
```